internal/cmd/request.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/server"
+	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/log"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+type CommonOptions struct {
+	Verbose bool `help:"Add the full request and response to the output. Be aware that the output will include extra details beyond the expected format." default:"false"`
+}
+
+type CustomFormat func(response core.APIResponse) string
```

Comment:
The name needs work.
You want all your variable names to be accurate and precise.
Here, your name is lying (not accurate).
This variable is not representing a "format", it is representing a "formatter".